Configuration/ProcessModifiers/python/beamSplash_cff.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
+
+# This modifier is for beam splash
+beamSplash =  cms.Modifier()
```

Comment:
is this meant to be picked up by Tier0 during splashes?  
If so, how do you envisage it, lacking an appropriate scenario?

Reply:
That is a good question. I made this PR in an attempt to address the issue reported on relval 138.3 . However I am not sure what would be the best strategy regarding its use in T0.